readme.txt

```diff
@@ -0,0 +1,8 @@
+----------------------------------------------------------
```

Comment:
Why do we have two readmes?

Reply:
I can delete that. Wasn't sure why we had that in the community template either.